include/oneapi/internal/math/blas_executor.h

```diff
@@ -215,12 +215,73 @@ class SyrkExecutor
     }
 };
 
+/**
+ *  <a name="DAAL-CLASS-ONEAPI-INTERNAL__AXPYEXECUTOR"></a>
+ *  \brief Adapter for AXPY routine
+ */
+class AxpyExecutor
+{
+private:
+    struct Execute
+    {
+        cl::sycl::queue & queue;
+        const uint32_t n;
+        const double a;
+        const UniversalBuffer& x_buffer;
+        const int incx;
+        const UniversalBuffer& y_buffer;
+        const int incy;
+        services::Status * status;
+
+        explicit Execute(cl::sycl::queue & queue, const uint32_t n, const double a, const UniversalBuffer& x_buffer, const int incx,
+               const UniversalBuffer& y_buffer, const int incy, services::Status * status = NULL)
+            : queue(queue),
+              n(n),
+              a(a),
+              x_buffer(x_buffer),
+              incx(incx),
+              y_buffer(y_buffer),
+              incy(incy),
+              status(status)
+        {}
+
+        template <typename T>
+        void operator()(Typelist<T>)
+        {
+            auto x_buffer_t = x_buffer.template get<T>();
+            auto y_buffer_t = y_buffer.template get<T>();
+
+            services::Status statusAxpy;
+#ifdef ONEAPI_DAAL_USE_MKL_GPU_FUNC
+            MKLAxpy<T> functor(queue);
+            statusAxpy = functor(n, (T)a, x_buffer_t, incx, y_buffer_t, incy);
+#else
+            // TODO
```

Comment:
Need to do. In the nightly build you will not be work

Reply:
done.